README.adoc

```diff
@@ -115,111 +114,6 @@ So for the time being, you need to:
 
 The https://github.com/input-output-hk/plutus-starter[plutus-starter] project provides an example.
 
-=== How to build the project's artifacts
```

Comment:
As for plutus-core, this info has been move to the relevant section in CONTRIBUTING.adoc